nebula-logger/core/main/logger-engine/classes/LogEntryEventBuilder.cls

```diff
@@ -1051,7 +1051,7 @@ global with sharing class LogEntryEventBuilder {
   private static void setUserInfoDetails(LogEntryEvent__e templateLogEntryEvent) {
     templateLogEntryEvent.Locale__c = System.UserInfo.getLocale();
     templateLogEntryEvent.LoggedById__c = System.UserInfo.getUserId();
-    templateLogEntryEvent.LoggedByUsername__c = System.UserInfo.getUserName();
+    templateLogEntryEvent.LoggedByUsername__c = System.UserInfo.getUsername();
```

Comment:
@jongpie, why? It was correct, according to the Salesforce API.
The IDE complains about the current one:
![image](https://github.com/user-attachments/assets/4a21559d-06c8-4b98-91f7-80e4222011d6)

Reply:
@TrangOul I've never seen that warning in VS Code, but maybe I've disabled whatever option that is - do you know what plugin/feature controls that warning message? 

Personally, I've always considered the user of `getUserName()` (capital `N`) a bug in Apex/Salesforce docs. I think `getUsername()` should be considered the correct casing - it's consistent with the rest of the platform, and it provides clarification that it is referring to the "username", and the "user's name".
- Everywhere else in Salesforce (and from what I've seen in other systems too/software engineering in general), it's always spelled `Username` with a lowercase `n`. The only place I'm aware of in Salesforce that uses the capital `N` is that method name. And since it returns the value of the field `User.Username`, I think the method name should use the same casing.
    - `User.Username` field has lowercase `n` in the field API name, the field label, and [the corresponding docs](https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_user.htm)
  
        ![image](https://github.com/user-attachments/assets/d2be7f36-e154-44bf-a6bf-faf6057e10d2)

    - Salesforce login pages (including Community Cloud/Experience Cloud, etc) all 
  
      ![image](https://github.com/user-attachments/assets/05989299-340b-413e-a558-8daa223a6e82)

I've also worked on multiple teams (at different companies) where using `getUserName()` is too ambiguous & confusing for people new to Apex (or anyone unfamiliar with the `UserInfo` class), occasionally resulting in bugs.
- Some people (incorrectly) thought `UserName` referred to the user's name (i.e., the `User.Name` field that combines `User.FirstName` and `User.LastName`
- Some people (correctly) thought `UserName` referred to the user's username (i.e., the `User.Username` field)

I think both are valid thoughts/guesses with the `UserName` casing - Salesforce uses `User` as a prefix on large number of standard fields (listed below), so I think there will always, inevitably, be some confusion if `getUserName()` refers to `User.Name` or `User.Username`
- `User.UserRoleId`
- `User.UserType`
- 12 fields that with `UserPermissions...`, like `User.UserPermissionsMobileUser`
- 81 fields that start with `UserPreferences....`, like `User.UserPreferencesUserDebugModePref`

But I've personally never had a situation where using `getUsername()` is unclear/confusing.

Reply:
@TrangOul thanks for the info & sharing your perspective! I might need to finally try Illuminated Cloud 2 again, it seems like it has a lot of cool features built in that I'd like to try out.

I wasn't aware of those inconsistencies of `sobj.getSObjectType()` and `recordId.getSobjectType()` 😭 That's kind of crazy, haha I wish Salesforce would be consistent (especially since `SObject` and `SObjectType` are a Salesforce-created terms). I personally find that having consistency (like always use `SObject`, never `Sobject`) really helps me with readability when jumping between different parts of the codebase (especially as Nebula Logger's codebase continues to grow in size/lines of code). But I totally understand your approach of sticking to what SF API uses if it simplifies your workflow.